README.md

```diff
@@ -134,6 +154,10 @@ import scala.concurrent.duration._
 object Main extends App {
   import ExecutionContext.Implicits._
 
+  case class Person(firstName: String, lastName: String)
```

Comment:
This works fine but, I think it points people towards a wrong patten. We should put the case class outside of the main object and create a companion object to the class person contain the codec. It's a little longer but it gives people the correct idea.